src/components/educationCard/EducationCard.js

```diff
@@ -16,6 +20,9 @@ export default function EducationCard({school}) {
       : null;
   };
   const {isDark} = useContext(StyleContext);
+
+  if (!school.logo)
+    console.log(`Image of schoolName ${school.name} is not available`);
```

Comment:
We can keep this in as it seems helpful but change it to `console.error('Image of ${school.name} is missing in education section');`